ddl/ddl_worker.go

```diff
@@ -623,6 +624,10 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 				return errors.Trace(err)
 			}
 
+			if job.IsQueueing() {
+				job.State = model.JobStateNone
```

Comment:
Could we remove this logic? We do `job.State = model.JobStateRunning` in `runDDL`.

Reply:
If we don't set it here, the state observed by the `OnJobRunBefore` callback would be "queueing". There are a few tests depend on this.

Reply:
I don't think it's necessary to change the code for testing. It's already in `OnJobRunBefore`, meaning `runDDL` hasn't been called yet, so I think the `queueing` is understandable.